src/styles/grade.scss

```diff
@@ -0,0 +1,26 @@
+.grade {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0 16px 24px;
+  font-size: 10px;
+  line-height: 14px;
+
+  &__stars {
+    display: flex;
+    justify-content: space-between;
+
+    &-star {
+      padding-right: 4px;
```

Comment:
The padding here is specified in `px`, but according to the [Code Style Rules](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules), it's recommended to use `rem` units for padding. Please convert it to `rem`.